src/gt4py/next/common.py

```diff
@@ -464,6 +464,9 @@ def dtype(self) -> core_defs.DType[core_defs.ScalarT]:
     def ndarray(self) -> core_defs.NDArrayObject:
         ...
 
+    def asnumpy(self) -> np.ndarray:
+        return np.asarray(self.ndarray)
```

Comment:
Should this be default implementation or just leave it blank?

Reply:
forgot to remove...